lib/resources.rb

```diff
@@ -111,13 +111,14 @@ def get_resource(key)
         end
       end
 
-      def authorization_types(provider: false, patient: false, system: false)
+      def authorization_types(provider: false, patient: false, practitioner: false, system: false)
```

Comment:
Provider/Practitioner any difference in Soarian? If so ignore this comment

Reply:
Removed as per Sudhir's comment